### DISCUSSION TITLE
[Week 25] Request for Contributions

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced/open-sauced#725] [Newsletter]
Starting this weekend, I plan to write up a list of projects looking for contributions. If you have a project that is looking for new contributors, please leave a comment below.  

# What is this? 👀
There are several projects looking for contributions from experience and first-time contributors. Below is a list of those projects.

### nteract (TypeScript/JavaScript/Python)
- [nteract](https://github.com/nteract/nteract/contribute) - 📘 The interactive computing suite for you! ✨
- [hydrogen](https://github.com/nteract/hydrogen/contribute). :atom: Run code interactively, inspect data and plot. All the power of Jupyter kernels, inside your favorite text editor.

### codeforPDX (Python)
- [recordexpungPDX](https://github.com/codeforpdx/recordexpungPDX/contribute) - A project to automate the expungement of qualifying criminal records. This project is done in conjunction with Qiu-Qiu Law.

### CockroachDB (Go)
- [cockroach](https://github.com/cockroachdb/cockroach/contribute) - CockroachDB - the open source, cloud-native distributed SQL database.

### webhint (TypeScript)
- [hint](https://github.com/webhintio/hint/contribute) - 💡 A hinting engine for the web

### WildAid (web, iOS, Android)
- WIP ([how to contribute](https://wildaid.github.io/contribute/)) - The WildAid Marine Program works to protect vulnerable marine environments.

**Comment by NotMoni** [comment]
Related: https://github.com/MunGell/awesome-for-beginners

**Comment by tannewt** [comment]
CircuitPython is a large project always looking for folks to join the community. https://circuitpython.org/contributing CircuitPython brings the ease of Python to hardware projects.

https://ElectionCal.us is a brand new project of mine to make election dates and deadlines easily shareable and accessible in a wide variety of formats. GitHub project is here: https://github.com/electioncal/us
